src/BusStop.js

```diff
@@ -1,54 +1,64 @@
-import { useState, useEffect } from 'react';
 
 function BusStop(props) {

```

Comment:
Instead of having props at the argument, it's helpful to have the unpacked object. It makes it more obvious to know what kind of things can be passed in.
For example:
`function BusStop({busStopData, stopName}) {`